test/integration/stomp_/publisher/create_request_test.py

```diff
@@ -43,47 +21,71 @@ def system_spec():
     }
 
 
-class MessageListener(object):
-    create_event_captured = False
+class MessageListener(ConnectionListener):
+    _create_event_captured: bool
+
+    def __init__(self):
+        super().__init__()
+        self._create_event_captured = False
 
     def on_error(self, headers, message):
         print("received an error %s" % headers)
 
-    def on_message(self, headers, message):
-        try:
-            if headers["model_class"] == "Operation":
+    def on_message(self, headers: Dict[str, Any], message: Any) -> None:
+        """Set a property when a MESSAGE frame is received by the STOMP connection.
 
-                parsed = SchemaParser.parse_operation(message, from_string=True)
+        If the needed key is in the headers dict and has the correct value, signal
+        that we've been successful.
 
-                if parsed.model and parsed.model.name:
-                    if parsed.model.name.startswith("REQUEST"):
-                        print(message)
-                        self.create_event_captured = True
+        Args:
+            headers: a dictionary containing all headers sent by the server
+            as key/value pairs.
+
+            message: the frame's payload - the message body.
+        """
+        try:
+            if "model_class" in headers:
```

Comment:
Though it's probably a fair assumption that this will never be called with an incorrect argument, this allows a more meaningful message to be produced if the passed dictionary doesn't have the key we're looking for.